### PR TITLE
Run 'composer install' in a docker builder stage

### PR DESCRIPTION
**Pull request by jkblume**
I wondered on the comment in the Dockerfile, which says to install the composer dependencies before building the image. 

Maybe it is an option for you to use a multi-stage Dockerfile, like i proposed in my PR :) I would appreciate to hear your thoughts about that.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,20 @@
 
+FROM composer AS builder
+
+COPY ./ /app
+RUN composer install
+
 # Running this container will start a language server that listens for TCP connections on port 2088
 # Every connection will be run in a forked child process
```
Comment (felixfbecker) [inline Dockerfile]: Can you keep this comment at the top?

**Review by felixfbecker — COMMENTED** [review]
Nice!

File: Dockerfile (hunk 1 above)
Comment (jkblume) [reply Dockerfile]: Yes of course :)

Comment (jkblume) [reply Dockerfile]: Moved the comments back to the top. I also replaced the deprecated maintainer with the appropriate "maintainer" label.

**Review by felixfbecker — APPROVED** [review]

**Pull request merged**